x-pack/plugins/ml/public/application/components/data_grid/feature_importance/decision_path_chart.tsx

```diff
@@ -88,38 +91,37 @@ export const DecisionPathChart = ({
   minDomain,
   maxDomain,
   baseline,
-  showValues,
 }: DecisionPathChartProps) => {
   // adjust the height so it's compact for items with more features
-  const baselineData: LineAnnotationDatum[] = useMemo(
-    () => [
-      {
-        dataValue: baseline,
-        header: baseline ? baseline.toPrecision(NUM_PRECISION) : '',
-        details: i18n.translate(
-          'xpack.ml.dataframe.analytics.explorationResults.decisionPathBaselineText',
-          {
-            defaultMessage:
-              'baseline (average of predictions for all data points in the training data set)',
-          }
-        ),
-      },
-    ],
+  const baselineData: LineAnnotationDatum[] | undefined = useMemo(
+    () =>
+      baseline && isRegressionFeatureImportanceBaseline(baseline)
```

Comment:
With Dima's suggestion making the type guard accept `any`, the `baseline &&` part here might then no longer be necessary.

Reply:
Updated here https://github.com/elastic/kibana/pull/80229/commits/10eae2d74c3867d9416aa3c29437da5d971cd485